### DISCUSSION TITLE
`Prompt` and `usePrompt` production ready?

### DISCUSSION THREAD

**Discussion by corymharper** [remix-run/react-router#10168] [Q&A] [answered]
The only feature blocking my organization from upgrading from v5 to v6 is the missing `Prompt` feature. I noticed the core functionality had been released via https://github.com/remix-run/react-router/issues/8139#issuecomment-1396078490. However I also notice they are prefixed with `unstable`. Should I be avoiding using these features in production?

**Comment by kiliman** [comment, marked as answer]
`unstable_` only means that the API may change, not that it's buggy. Since it's something you need, it would help the team if you tried it in your app and provided feedback.
